test/integration-tests/run-tests.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Cisco and/or its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VPP_IMAGE="calicovpp/vpp:latest"
+POD_MOCK_IMAGE="calicovpp/vpp-test-pod-mock:latest"
+
+function prepageImageWithVPPBinary() {
+  if ! docker image inspect $VPP_IMAGE >/dev/null 2>&1; then
+    echo "Can't find docker image "$VPP_IMAGE". Rebuilding it..."
+    pushd ../../vpp-manager
+    if ! make image; then
+      echo "Can't build "$VPP_IMAGE", exiting..."
+      exit 1
+    fi
+    popd
+  fi
+}
+
+function preparePodMockImage() {
+  if ! docker build -t $POD_MOCK_IMAGE images/pod-mock; then
+      echo "Can't build "$POD_MOCK_IMAGE", exiting..."
+      exit 1
+  fi
+}
+
+result=0
+
+echo "Running Integration tests..."
+echo "Running Calico VPP Agent - CNI tests..."
+prepageImageWithVPPBinary
+preparePodMockImage
+# Note: some pod tests expect elevated user privileges -> using sudo
+INTEGRATION_TEST="." VPP_IMAGE=$VPP_IMAGE VPP_BINARY="/usr/bin/vpp" \
+sudo -E env "PATH=$PATH" go test -v -run Integration ../../calico-vpp-agent/cni -ginkgo.v || result=$?
```

Comment:
The ideal setup in which to run the tests is still a bit unclear to me. In the context of a CI, we'd probably want to run everything in a dedicated container (so without sudo) but for development it's probably worth having the option to sudo it...